lib/atp_scraper/activities/record.rb

```diff
@@ -0,0 +1,24 @@
+module Activities
+  # Activity Record Class
+  class Record
+    def self.pickup_record(record_doc)
+      result = {}
+      record_doc.css("td").each_with_index do |td, n|
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'n' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)